app/components/dialogs/NewBoardDialog.tsx

```diff
@@ -136,6 +158,27 @@ export default function NewBoardDialog(props: NewBoardDialogProps) {
             </Select>
           </FormControl>
         </div>
+        <div className={classes.formControl}>
+          <FormControl variant="outlined" fullWidth>
+            <InputLabel id="demo-simple-select-filled-label">
+              Create from old board
```

Comment:
Create from existing board sounds better. You already have it in the label of Select.